### ISSUE TITLE
Title attribute?

### ISSUE THREAD

**Issue by mbostock** [observablehq/inputs#68]
It could be useful in a few contexts.

**Comment by hellonearthis** [comment]
Yes,  I would like to see a way to change the titles as the datasets names are not always good to humans.
~~~js
title: { 'geography_name':'Region','no3_kg_per_yr', 'Nitrates (kg per year)' }
~~~

**Comment by mbostock** [comment]
@hellonearthis This issue is for the basic inputs (e.g., a button or checkbox), where the *title* option would be specified as a string. I’m guessing that your comment is with regard to the Table component as a way to format the column header. You can rename the columns currently by using *array*.map on your data, e.g.,

```js
Table(data.map(d => ({"Region": d. geography_name, "Nitrates (kg per year)": d.no3_kg_per_yr}))
```

That said, it’d be nice if Table had a way to express derived columns without needing to materialize all the objects. I’ll file a separate issue for that.
